app-web/__mocks__/got.js

```diff
@@ -0,0 +1,6 @@
+jest.requireActual('got');
```

Comment:
I think you need all your files to have the standard disclaimer blerb at the top. I don't recall the history but I was advised to do it by Todd when I started and he's quite knolidable about licencing best practices.